oasis/projects.md

```diff
@@ -50,16 +50,16 @@ Deleting a project will delete contained deployments, certificates & IP allowlis
 This operation is **irreversible**.
 {% endhint %}
 
-1. Click on _Overview_ in the _Projects_ section of the main navigation.
-2. Click on a trash can icon in the _Actions_ column.
-3. Enter `Delete!` to confirm and click _Yes_.
+1. Click __Overview__ in the __Projects__ section of the main navigation.
+2. Click a trash can icon in the __Actions__ column.
```

Comment:
Please replace all occurrences of the "trash" and "trash can" with "the recycle bin"